docs/devguide/create-metricset.asciidoc

```diff
@@ -242,12 +242,20 @@ var (
 )
 
 func eventMapping(input map[string]interface{}) common.MapStr {
-	return schema.Apply(input)
+	return schema.Apply(input) <4>
 }
 ----
+<1> Marks a field as required.
+<2> If a field has no schema option set, it is equivalent to `Required`.
+<3> Marks the field as optional.
```

Comment:
@odacremolbap added a new option to ignore all errors of an specific field (https://github.com/elastic/beats/pull/12089), it may worth to mention it here too.

Reply:
`IgnoreAllErrors` option, it is aded by #12089